source/install.txt

```diff
@@ -108,22 +105,46 @@ Select the appropriate tab for your operating system:
          .. tab:: .deb
             :tabid: deb
 
+            Supported Platforms
+            ~~~~~~~~~~~~~~~~~~~
+
             :binary:`mongosh` is available as a :abbr:`PPA (Personal
-            Package Archive)` for Ubuntu 20.04 (Focal) and Ubuntu 18.04
-            (Bionic).
+            Package Archive)` for the following platforms:
+            
+            - Ubuntu 20.04 (Focal)
+            - Ubuntu 18.04 (Bionic)
+
+            Procedure
+            ~~~~~~~~~
 
             .. include:: /includes/steps/install-shell-linux-deb.rst
 
          .. tab:: .rpm
             :tabid: rpm
+
+            Supported Platforms
+            ~~~~~~~~~~~~~~~~~~~
             
-            :binary:`mongosh` is available as ``yum`` package for
-            :abbr:`RHEL (Red Hat Enterprise Linux)` and Amazon Linux 2.
+            :binary:`mongosh` is available as ``yum`` package for the
+            following platforms:
+
+            - :abbr:`RHEL (Red Hat Enterprise Linux)`
+            - Amazon Linux 2
+
+            .. important:: Amazon Linux 2023 Support
+
+               ``mongosh`` is **not** supported on Amazon Linux 2023.
```

Comment:
Just to avoid any confusion – mongosh *is* supported on Amazon Linux 2023. It’s the PPA that is missing temporarily.